guides/micronaut-rabbitmq-rpc/bookrecommendation/groovy/src/main/groovy/example/micronaut/Book.groovy

```diff
@@ -0,0 +1,23 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+@CompileStatic
+@EqualsAndHashCode
+@ToString
+class Book {
+
+    final String isbn
+    final String name
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.